packages/hash/api/src/graphql/typeDefs/ontology/entity-type.typedef.ts

```diff
@@ -0,0 +1,56 @@
+import { gql } from "apollo-server-express";
+
+export const entityTypeTypedef = gql`
+  scalar EntityType
+
+  type PersistedEntityType {
+    entityTypeVersionedUri: String!
+    createdBy: ID!
```

Comment:
I think we should be consistent about what we name these fields across the GQL API. Since you're referring to this field using `accountId` for the other queries related to entity types, I think we should use `accountId` here for now:
```suggestion
    accountId: ID!
```

If you agree, I think we would need to change this for the other GQL types as well (`PersistedPropertyType` and `PersistedDataType`). If you want to keep `createdBy` I'm wondering what would happen if a user creates an entity type in another account (for example the one of an organisation)? And we should probably use a `createdBy` argument for the other mutations/queries related to types?

Reply:
Addressed in 9d88b81b307473c8c5fbbc21f34af7c828a358af for all types. I took the liberty to also rename `schema` to the respective kind of type.

Reply:
Great, thanks :)